plugins/cck_field/select_dynamic/tmpl/edit.php

```diff
@@ -79,7 +79,7 @@
 	$('#json_options2_table, #json_options2_name, #json_options2_value, #json_options2_where, #json_options2_orderby, #json_options2_limit, #json_options2_attr1, #blank_li').isVisibleWhen('bool2','0');

```

Comment:
Klas,

$('#json_options2_table, #json_options2_name, #json_options2_value, #json_options2_where, #json_options2_orderby, #json_options2_limit, #blank_li').isVisibleWhen('bool2','0');

This line should be like this, without the '#json_options2_attr1' in the list of fields to show if #bool2=0. This is what really causes the incorrect behavior with the custom attributes toggle.